exercises/concept/black-jack/black_jack_test.py

```diff
@@ -1,43 +1,109 @@
 import unittest
 import pytest
 from black_jack import (
-    number_of_card,
-    number_of_ace,
-    blackjack
+    value_of_card,
+    value_of_ace,
+    is_blackjack,
+    can_split_pairs,
+    can_double_down
 )
 
 
-class TestComparisons(unittest.TestCase):
+class TestBlackJack(unittest.TestCase):
 
     @pytest.mark.task(taskno=1)
-    def test_number_of_card(self):
-        input_data = ['K', 'A']
-        output_data = [10, "ace"]
-        number_of_variants = range(1, len(input_data) + 1)
+    def test_value_of_card(self):
+        data = [
+            ('2', 2),
+            ('5', 5),
+            ('8', 8),
+            ('10', 10),
+            ('J', 10),
+            ('Q', 10),
+            ('K', 10),
+        ]
 
-        for variant, input, output in zip(number_of_variants, input_data, output_data):
-            with self.subTest(f"variation #{variant}", input=input, output=output):
-                self.assertEqual(number_of_card(input[0], input[1]), output,
-                                 msg=f'Expected: {output} but the number of cards was calculated incorrectly.')
+        for variant, (card, value) in enumerate(data, 1):
```

Comment:
Nice work here.  This is much cleaner.  Makes me want to go back and change my test files.